app/internal/page/address_form.go

```diff
@@ -46,11 +46,11 @@ func (f *addressForm) Validate() validation.List {
 
 	switch f.Action {
 	case "lookup":
-		errors.String("lookup-postcode", "postcode", f.LookupPostcode,
+		errors.String("lookup-postcode", "aPostcode", f.LookupPostcode,
```

Comment:
are we going to want a ticket to add "a/an" for all field error messages where it makes sense?

Reply:
Yeah, worth running through with Amy and Sophie on recent validation error changes so we can hit them up in one go.